lib/spack/spack/cmd/diy.py

```diff
@@ -62,6 +68,10 @@ def diy(self, args):
     if not args.spec:
         tty.die("spack diy requires a package spec argument.")
 
+    if args.jobs is not None:
```

Comment:
Since this is a common argument with `install`, can you factor this and arg from `install` into [`spack.cmd.common.arguments`](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/cmd/common/arguments.py)?  Look at the `find`, `install`, `module`, `setup` and `spec` commands do it for examples.